.github/workflows/ci.yml

```diff
@@ -7,6 +7,16 @@ on:
         branches: master
 
 jobs:
+    Shell-check:
```

Comment:
`shellcheck` is another linter, like flake8... So let's use `Lint-shellcheck` as the visible name.

Reply:
Done 👍